src/odemis/acq/fastem_conf.py

```diff
@@ -86,6 +87,8 @@ def configure_scanner(scanner, mode):
     scanner.multiBeamMode.value = conf["multiBeamMode"]
     scanner.external.value = conf["external"]
     scanner.blanker.value = conf["blanker"]
+    if "dwellTime" in kwargs:
+        scanner.dwellTime.value = kwargs["dwellTime"]
```

Comment:
Why do you have to do it here? Is there something special related to the "external" VA? If so, it should be explained. But alternatively, it should be fixed in the driver.

Reply:
> But alternatively, it should be fixed in the driver.

This sound like a better fix, I will discuss it with @tepals